src/store/accounts.rs

```diff
@@ -95,12 +95,15 @@ impl Store {
         self.db
             .prepare(QUERY)
             .map_err(StoreError::QueryError)?
-            .query_map([], |row| row.get(0))
+            .query_map([], |row| {
+                let row: i64 = row.get(0)?;
+                Ok(row)
+            })
             .expect("no binding parameters used in query")
             .map(|result| {
                 result
                     .map_err(StoreError::ColumnParsingError)
-                    .map(|id: u64| AccountId::try_from(id).expect("account id is valid"))
+                    .map(|id: i64| AccountId::try_from(id as u64).expect("account id is not valid"))
```

Comment:
Just to be clear, in the Miden projects (not sure but it could be the norm as well) the `expect()` messages is usually why you expect the statement to be valid. For instance:

```
        let timestamp = SystemTime::now()
            .duration_since(UNIX_EPOCH)
            .expect("we are after 1970")
            .as_millis() as u64;
            ```

Reply:
Oh, I thought it was a typo. Thanks for the clarification